pkg/cli/testdata/zip/testzip_external_process_virtualization

```diff
@@ -85,9 +85,7 @@ debug zip --concurrency=1 --cpu-profile-duration=1s /dev/null
 [cluster] retrieving SQL data for system.tenant_tasks... writing output: debug/system.tenant_tasks.txt... done
 [cluster] retrieving SQL data for system.tenant_usage... writing output: debug/system.tenant_usage.txt... done
 [cluster] retrieving SQL data for system.tenants... writing output: debug/system.tenants.txt... done
-[cluster] requesting nodes... received response...
-[cluster] requesting nodes: last request failed: rpc error: ...
-[cluster] requesting nodes: creating error output: debug/nodes.json.err.txt... done
+[cluster] requesting nodes... received response... writing JSON output: debug/nodes.json... done
```

Comment:
@stevendanna - sorry for the noise on this PR. After pushing the changes I notices a couple of test failures in the CI run. They are related to data driven tests. I fixed the expected zip log to be inline with the code fixes done. 

More importantly, I was curious why `testzip_external_process_virtualization` test was not failing with capability missing error. After carefully looking at both the test for differences, data driven tests in `zip_tenant_test.go` are running in insecure mode due to https://github.com/cockroachdb/cockroach/issues/77173 and that's the reason they were passing without any issues. 
https://github.com/cockroachdb/cockroach/blob/c68c559859be738efead9971f5e11f62a8c69d06/pkg/cli/zip_tenant_test.go#L76-L86 
I'll add this issue to CRDB-38970 it is not already being tracked. This PR is ready to be committed now. Can you please take a look and bless it one more time?